### PR TITLE
Effigy Labs PID Request

### PR DESCRIPTION
**Pull request by astrojr1**
Hello, requesting 2 PIDs under 1209, EFFA, and EFFE, one for bootloader, one for main function.  We believe we have complied with the PID request instructions, however, please let us know if not.  Thank you for your time.

### CODE REVIEW

File: 1209/EFFA/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: EFFA
```
Comment (peternewman) [inline 1209/EFFA/index.md]: This should be "layout: pid" (without the quotes).

File: 1209/EFFE/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: EFFE
```
Comment (peternewman) [inline 1209/EFFE/index.md]: Again

**Comment by peternewman** [comment]
@Arachnid this could do with a review/merge too.

**Comment by astrojr1** [comment]
Hi, any problems?

**Pull request merged**